projects/1.20/assets/ad-astra/ad_astra/lang/zh_cn.json

```
@@ -324,23 +395,77 @@
   "block.ad_astra.venus_stone_bricks": "金星石砖",
   "block.ad_astra.venus_stone_slab": "金星石台阶",
   "block.ad_astra.venus_stone_stairs": "金星石楼梯",
-  "block.ad_astra.wall_extinguished_torch": "墙上的熄灭的火把",
   "block.ad_astra.water_pump": "水泵",
   "block.ad_astra.white_flag": "白色旗帜",
+  "block.ad_astra.white_industrial_lamp": "白色工业灯",
   "block.ad_astra.yellow_flag": "黄色旗帜",
+  "block.ad_astra.yellow_industrial_lamp": "黄色工业灯",
+  "config.ad_astra.allowFlagImages": "允许在旗帜上显示图像",
+  "config.ad_astra.atmosphereLeave": "离开大气层",
+  "config.ad_astra.client.title": "Ad Astra客户端",
+  "config.ad_astra.clientConfig": "客户端配置文件",
+  "config.ad_astra.coalGeneratorEnergyGenerationPerTick": "燃煤发电机每Tick产能速率",
+  "config.ad_astra.description": "生命长寿而繁荣，愿原力与你同在。",
+  "config.ad_astra.deshTier": "戴斯等级",
+  "config.ad_astra.disableAirVortexes": "禁用气旋",
+  "config.ad_astra.disableGravity": "关闭重力",
+  "config.ad_astra.disableOxygen": "关闭氧气",
+  "config.ad_astra.disableTemperature": "关闭温度",
+  "config.ad_astra.disabledPlanets": "禁用星球",
+  "config.ad_astra.distributionRefreshRate": "传输刷新速率",
+  "config.ad_astra.energizerEnergyCapacity": "充能器能量容量",
+  "config.ad_astra.etrionicBlastFurnaceBlastingEnergyPerItem": "Etrionic高炉每烧炼一个物品消耗能量",
+  "config.ad_astra.forcePlanetTick": "强制星球刻",
+  "config.ad_astra.ironTier": "铁等级",
+  "config.ad_astra.launchEfficientFuelCost": "发射高效燃料消耗",
+  "config.ad_astra.launchFromAnywhere": "从任意地点发射",
+  "config.ad_astra.launchFuelCost": "发射燃料消耗",
+  "config.ad_astra.machine.energyCapacity": "能量容量",
+  "config.ad_astra.machine.fluidCapacity": "流体容量",
+  "config.ad_astra.machine.maxEnergyInOut": "最大能量输入/输出",
+  "config.ad_astra.machines": "机器",
+  "config.ad_astra.maxDistributionBlocks": "最大散布方块距离",
+  "config.ad_astra.ostrumTier": "紫金等级",
+  "config.ad_astra.pipeRefreshRate": "管道刷新速率",
+  "config.ad_astra.planetRandomTickSpeed": "星球随机刻速度",
+  "config.ad_astra.steelTier": "钢等级",
+  "config.ad_astra.title": "Ad Astra",
+  "config.ad_astra.waterPumpEnergyPerTick": "水泵每Tick能量消耗",
+  "config.ad_astra.waterPumpFluidGenerationPerTick": "水泵每Tick流体抽取速率",
   "death.attack.acid_rain": "%1$s被酸雨腐蚀了",
-  "death.attack.cryo_fuel": "%1$s冻成了冰块",
-  "death.attack.oxygen": "%1$s缺氧窒息而亡",
-  "death.attack.oxygen.player": "%1$s缺氧窒息而亡",
+  "death.attack.acid_rain.player": "%1$s在尝试逃脱%2$s时被酸雨腐蚀了",
```

```suggestion
  "death.attack.acid_rain.player": "%1$s在试图逃离%2$s时被酸雨腐蚀了",
```
https://zh.minecraft.wiki/w/%E6%AD%BB%E4%BA%A1%E6%B6%88%E6%81%AF